Subject: [PATCH] ext4: remove duplicated declarations in inode.c

From: Zheng Liu <wenqing.lz@taobao.com>

in patch cb20d5188366f04d96d2e07b1240cc92170ade40, ext4_set_bh_endio and
ext4_end_io_buffer_write are declared at the beginning of inode.c file.  So we
can remove duplicated declarations in this file.

Signed-off-by: Zheng Liu <wenqing.lz@taobao.com>
---
 fs/ext4/inode.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/fs/ext4/inode.c b/fs/ext4/inode.c
index 02bc8cb..dd0fa3c 100644
--- a/fs/ext4/inode.c
+++ b/fs/ext4/inode.c
@@ -1925,9 +1925,6 @@ out:
 	return ret;
 }
 
-static int ext4_set_bh_endio(struct buffer_head *bh, struct inode *inode);
-static void ext4_end_io_buffer_write(struct buffer_head *bh, int uptodate);
-
 /*
  * Note that we don't need to start a transaction unless we're journaling data
  * because we should have holes filled from ext4_page_mkwrite(). We even don't
-- 
1.7.4.1

